### PR TITLE
Ws3li caption

### LINKED ISSUE #13: ws3li checklist
- [ ] Create a branch
- [ ] Make changes
- [ ] Make a commit
- [ ] Push our changes
- [ ] Review a PR
- [ ] Merge

### PR DESCRIPTION
**Pull request by ws3li**
Fixes the default caption to something unique.

Closes #13

### CODE REVIEW

File: _posts/0000-01-06-ws3li.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+:wave: Private Investigation Cat
```
Comment (beardofedu) [inline _posts/0000-01-06-ws3li.md]: Perhaps change this to `Detectocat`

Comment (ws3li) [reply _posts/0000-01-06-ws3li.md]: nah I like it like this. Thanks thou!

File: _posts/0000-01-06-ws3li.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: slide
...
 
-![skitchtocat](https://octodex.github.com/images/skitchtocat.png)
+![privateinvestocat](https://octodex.github.com/images/privateinvestocat.jpg)
```
Comment (beardofedu) [inline _posts/0000-01-06-ws3li.md]: lol example msg

Comment (beardofedu) [inline _posts/0000-01-06-ws3li.md]:
```suggestion
![skateocat](https://octodex.github.com/images/skatetocat.png)
```

What about using this instead?

**Pull request merged**